tools/packaging/Makefile

```diff
@@ -10,9 +10,9 @@ SED := sed
 YQ := $(MK_DIR)/yq
 SNAPCRAFT_FILE := snap/snapcraft.yaml
 VERSIONS_YAML_FILE := versions.yaml
-VERSIONS_YAML_FILE_URL := https://raw.githubusercontent.com/kata-containers/runtime/master/versions.yaml
+VERSIONS_YAML_FILE_URL := https://raw.githubusercontent.com/kata-containers/kata-containers/2.0-dev/versions.yaml
 VERSION_FILE := VERSION
-VERSION_FILE_URL := https://raw.githubusercontent.com/kata-containers/runtime/master/VERSION
+VERSION_FILE_URL := https://raw.githubusercontent.com/kata-containers/kata-containers/2.0-dev/VERSION
```

Comment:
We should just use `../../versions.yaml` instead of trying to download it from github.

Reply:
@bergwolf   ok,  i  recommit a new patch.

Reply:
@bergwolf  Maybe the "snap" target in Makefile  is no longer used, but i not sure, so it can be deleted or updated in another issue and pr.